doc/source/myrocks/variables.rst

```diff
@@ -2174,7 +2174,25 @@ for the MyRocks MTR test suites.
 
 Specifies the level of information to capture with the Perf Context plugins.
 Default value is ``0``.
-Allowed range is up to ``4``.
+Allowed range is up to ``5``.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Value
+     - Description
+   * - 0
```

Comment:
It is an interesting setting. "Unknown". In implementation it acts as 'disable (1)', but when the value of the variable is queried it indeed returns unknown (0). Moreover, unknown is the default...

Why just don't have values from 1(disabled (default)) to 5?

Reply:
Discussed in Slack.